index.html

```diff
@@ -0,0 +1,322 @@
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+        <div class="suggestions">
+          <iframe
+            width="150"
+            height="96"
+            src="https://www.youtube.com/embed/V74l_zS1x8E"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+            allowfullscreen
+          ></iframe>
+          <p>video 1</p>
+        </div>
+      </section>
+      <section class="comments">
+        <h1>Video Title</h1>
+        <div class="channelinfo">
+          <a href="#"><img src="./assets/channels4_profile.jpg" /></a>
+          <a href="#">Channel name</a>
+          <a href="#">Subscribe</a>
+        </div>
+        <hr />
+        <DIV class="commentsec">
```

Comment:
Kindly check your casing for the Div. This is to make sure you follow the correct html semantic tags. Please use <div> not <DIV>. That is the correct way to name your tags. ;)